src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/host/host.module.ts

```diff
@@ -0,0 +1,15 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { HostsComponent } from './hosts/hosts.component';
+import { SharedModule } from '../../shared/shared.module';
+import { ServiceListPipe } from './service-list.pipe';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    SharedModule
+  ],
+  declarations: [HostsComponent, ServiceListPipe],
+})
+export class HostModule { }
```

Comment:
"Host" module should be renamed to "Cluster", to match the existing ceph dashboard structure.

Reply:
Done